traitsui/wx/led_editor.py

```diff
@@ -59,18 +53,12 @@ def update_editor(self):
         """
         self.control.SetValue(self.str_value)
 
+    def _alignment_changed(self):
+        if self.control is not None:
+            self.control.SetAlignment(LEDStyles[self.alignment])
 
-# -------------------------------------------------------------------------
-#  Create the editor factory object:
-# -------------------------------------------------------------------------
-
-# wxPython editor factory for LED editors:
-
-
-class LEDEditor(BasicEditorFactory):
-
-    #: The editor class to be created:
-    klass = _LEDEditor
 
-    #: The alignment of the numeric text within the control:
-    alignment = Enum("right", "left", "center")
+# editor names for factory to find
+ReadonlyEditor = LEDEditor
+SimpleEditor = LEDEditor
+CustomEditor = LEDEditor
```

Comment:
Do we actually need to populate all the three styles?  If the custom/readonly editors are not implemented, then in the future we will be certain that they are never used because attempts to set `style='custom'` or `style='readonly'` will be met with an error (I think). If we populate them, it will add constraint to future extensions when we do want the custom / readonly style to behave completely differently.

e.g. The TableEditor does not have a custom editor. ButtonEditor does not have a readonly editor. They don't seem to be missed.